.github/workflows/slow.yaml

```diff
@@ -223,13 +223,13 @@ jobs:
           run: |
             export MAKE=gmake
             export pjobs="-j`gnproc`"
-            export AUTOMAKE_VERSION=1.16
+            export AUTOMAKE_VERSION=`ls /usr/local/bin/automake-* | sed 's/.*-//' | tail -1`
             export amver=${AUTOMAKE_VERSION}
             export ACLOCAL_AUTOMAKE_DIR="/usr/local/share/aclocal-${AUTOMAKE_VERSION}"
             export ACLOCAL_PATH="/usr/local/share/aclocal:/usr/local/share/aclocal-${AUTOMAKE_VERSION}"
-            export AUTOCONF_VERSION=2.72
+            export AUTOCONF_VERSION=`ls /usr/local/bin/autoconf-* | sed 's/.*-//' | tail -1`
             export acver=${AUTOCONF_VERSION}
-            export ltver=2.4.2
+            export ltver=`libtool --version | head -1 | sed 's/.* //'`
```

Comment:
The fact that `libtool --version` works without specifying a version number in the `libtool` binary name implies that our `./bootstrap.sh` should be able to correctly locate the `libtoolize` binary without setting this `ltver` variable.

Please check if that above is correct and, if possible remove this part of the setup.

Reply:
removed